### PR TITLE
Wordwrap all lines to 80 characters width by default

### LINKED ISSUE #4104: Help doesn't wordwrap very well.
If you do for instance `wp help media regenerate` then the output is poorly wordwrapped:

screenshot from 2017-05-30 13-15-40

A fix is to do one wordwrap near the end instead of several as is done now, see https://github.com/gitlost/wp-cli/pull/6. I'll do a PR.

### PR DESCRIPTION
**Pull request by gitlost**
Fixes #4104

Does one wordwrap near the end.

It adds a new global parameter `help_wordwrap_width` which I'm not too sure is warranted - maybe just use 90 though it seems pretty arbitrary?

### CODE REVIEW

File: php/config-spec.php [hunk 1]
```diff
@@ -105,6 +105,14 @@
 		'default' => array(),
 	),
 
+	'help_wordwrap_width' => array(
```
Comment (danielbachhuber) [inline php/config-spec.php]: This should be an environment variable, not a global parameter.

Global parameters conflict with local parameters (see #1939 for an example). At this point, we pretty much should never add a new global parameter.

**Comment by danielbachhuber** [comment]
Does this / should this wordwrap examples?

> maybe just use 90 though it seems pretty arbitrary?

iirc 80 is a more common standard.

File: php/config-spec.php (hunk 1 above)
Comment (gitlost) [inline php/config-spec.php]: Ah I see - changed to env var.

**Comment by gitlost** [comment]
Changed to use environment variable and default to 80 (though it's a bit narrow - I think it might be more a 100 chars is the norm now).

**Comment by gitlost** [comment]
> Does this / should this wordwrap examples?
Yes, added to tests.

**Pull request merged**